src/hangar/cli/cli.py

```diff
@@ -175,15 +176,22 @@ def create_arrayset(repo: Repository, name, dtype, shape, variable_, named):
 
           $ hangar arrayset create --variable-shape train_images UINT8 256 256 3
 
+    To specify that the column contains a nested set of subsample data under a
+    common sample key, the ``--contains-subsamples`` flag can be used.
+
+       .. code-block:: console
+
+          $ hangar arrayset create --contains-subsamples train_images UINT8 256 256 3
+
     """
     try:
         co = repo.checkout(write=True)
         aset = co.arraysets.init_arrayset(name=name,
                                           shape=shape,
                                           dtype=np.typeDict[dtype.lower()],
-                                          named_samples=named,
-                                          variable_shape=variable_)
-        click.echo(f'Initialized Arrayset: {aset.name}')
+                                          variable_shape=variable_,
+                                          contains_subsamples=subsamples_)
+        click.echo(f'Initialized Arrayset: {aset.arrayset}')
```

Comment:
IMO `name` was more intuitive than `arrayset`

Reply:
yeah, but when we update the internal naming from `arrayset ->columns`, this is going to have to be more descriptive than `name`.  I'm actually going to break this `NamedTuple` class into `column-type` specific `NamedTuple` classes. It's going to be messy to refactor, this will just help in the future (it's a temporary name).